src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -226,7 +228,8 @@
                                         (get-in route [:from :native-currency-symbol]))
             native-token              (when native-currency-symbol
                                         (rf/sub [:wallet/token-by-symbol native-currency-symbol]))
-            fee-in-native-token       (when-not confirm-disabled? (send-utils/calculate-gas-fee route))
+            fee-in-native-token       (when-not confirm-disabled?
+                                        (reduce + (map send-utils/calculate-gas-fee route)))
```

Comment:
why would we not move this to that utils and abstract away?

Reply:
Yep we can, I'll move it